test/unit_test/rag/utils/test_extract_html_headers.py

```
@@ -0,0 +1,44 @@
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+from rag.utils import file_utils
+
+
+class _Response:
+    content = b"<html />"
+    url = "https://example.test/"
+    headers = {"Content-Type": "text/html"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)